ihatemoney/tests/conftest.py

```diff
@@ -0,0 +1,60 @@
+from unittest.mock import MagicMock
+
+from flask import Flask
+import pytest
+
+from ihatemoney.currency_convertor import CurrencyConverter
+from ihatemoney.run import create_app, db
+
+
+@pytest.fixture
+def app(request: pytest.FixtureRequest):
+    """Create the Flask app with database"""
+    app = create_app(request.cls)
+
+    with app.app_context():
+        db.create_all()
+    request.cls.app = app
+
+    yield app
+
+    # clean after testing
+    db.session.remove()
+    db.drop_all()
+
+
+@pytest.fixture
+def client(app: Flask, request: pytest.FixtureRequest):
+    client = app.test_client()
+    request.cls.client = client
+
+    yield client
+
+
+@pytest.fixture
+def app_ctx(app):
+    """
+    This fixture add both app_context AND request context for ease of use.
+    If you only need an app_context locally, use `with self.app.app_context():`
+    in your code.
+    """
+    with app.app_context(), app.test_request_context():
+        yield
+
+
+@pytest.fixture
```

Comment:
I'm not sure if it's possible with pytest, but any chance this could be in the same file as the tests?

Reply:
`conftest.py` is the place for shared fixtures, but pytest allows to define fixtures in test files :ok_hand: